tests/drawing_functions_test.py

```diff
@@ -13,13 +12,7 @@ def setUp(self):
         super(TestDrawingFunctions, self).setUp()
         self.w = World.open_protobuf("%s/seed_28070.world" % self.tests_data_dir)
 
-    def test_draw_ancient_map_factor1(self):
-        w_large = World.from_pickle_file("%s/py%s_seed_48956.world" % (self.tests_data_dir, platform.python_version_tuple()[0]))
-        target = PNGWriter.rgba_from_dimensions(w_large.width, w_large.height)
-        draw_ancientmap(w_large, target, resize_factor=1)
-        self._assert_img_equal("ancientmap_48956", target)
-
-    def test_draw_ancient_map_factor3(self):
+    def test_draw_ancient_map(self):
```

Comment:
One was testing a pickle-file, one a protobuf-file. They did pretty much the same anyway. Should another format need a test like this, maybe it should get it's own `drawing_functions_test.py`?